mas-foundation/src/main/java/com/ca/mas/core/util/FileUtils.java

```diff
@@ -0,0 +1,35 @@
+package com.ca.mas.core.util;
+
+
+import com.ca.mas.foundation.MASException;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * Utility methods for file operations.
+ */
+public class FileUtils {
+
+    /**
+     * Returns the file content as byte array.
+     *
+     * @param path the absolute path of file
+     * @return file content as byte array
+     */
+    public static byte[] getBytesFromPath(String path) throws MASException {
+        byte[] bytes = {};
+        try {
+            File file = new File(path);
+            bytes = new byte[(int) file.length()];
+            InputStream is = new FileInputStream(file);
+            is.read(bytes);
+            is.close();
```

Comment:
Never close any stream from inside the try code. In case is.readBytes  errors out then the Stream will never be closed. Do that in a finally block.

Reply:
Solved.